server/controllers/thoughtController.js

```diff
@@ -0,0 +1,127 @@
+const User = require("../model/User");
+const Thought = require("../model/Thought");
+
+const addThought = async (req, res) => {
+    try {
+        const thought = await Thought.create(req.body);
+        const user = await User.findOneAndUpdate(
+            { username: req.body.username },
+            { $addToSet: { thoughts: thought._id } },
+            { new: true }
+        )
+        res.status(200).json({ user, thought });
+    } catch (err) {
+        console.log(err);
+        res.status(500).json(err);
+    }
+}
+
+const getAllThoughts = async (req, res) => {
```

Comment:
LOVE that you've changed the code base to be written according to the ES6 version. Really appreciate you taking the time to rewrite the functions into arrow functions, sticking to async/await and encompassing all your logic in try/catch blocks - primary example of a good code base :-)